code/style.css

```diff
@@ -2,6 +2,491 @@
+.heroPic img {
+  width: 100%;
+  background-position: center;
+  background-repeat: none;
+  background-size: cover;
+}
+
+.hero-ctn {
+  text-align: center;
+  width: 100%;
+}
+
+.heroText {
+  position: relative;
+  box-sizing: border-box;
+  font-size: 35px;
+  padding: 30px;
+  margin-top: 20px; 
+}
+
+
+/* Media queries for hero text */
+@media only screen and (min-width: 768px) {
+
+  .heroText {
+    font-size: 50px;
+  }
+} 
+
+/* Media queries for hero text desktop */
+@media only screen and (min-width: 992px) {
+
+  .heroText{
+    position: absolute;
+    z-index: 1;
+    top: 25%;
+    font-size: 50px;
+    text-shadow: 5px 5px  #ffffff;
+    color: #0F1B12;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+}
+
+/* Media queries for hero text big screen */
+@media only screen and (min-width: 1056px) {
+  .heroText{ top: 30%;
+  font-size: 70px;
+  }
+}
+
+
+/* Form */
+.signUpContent {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 30px;
+  justify-self: center;
+  max-width: 1400px;
+}
+
+.signUpForm {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.signUpForm h2 {
+  font-weight: 400;
+  font-size: 20px;
+  color: #0F1B12;
+}
+
+form label {
+  text-align: left;
+  font-size: 14px;
+  box-sizing: border-box;
+  padding: 20px 0px 5px;
+}
+
+.excitedVolLabel {
+  margin-bottom: 20px;
+}
+
+.excitedVol {
+  -webkit-appearance: none;
+  width: 100%;
+  height: 10px;
+  border-radius: 20px;
+  background: #d3d3d3;
+  outline: none;
+  opacity: 0.7;
+  -webkit-transition: .2s;
+  transition: opacity .2s;
+  margin-bottom: 10px;
+}
+
+.excitedVol::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: #4CAF50;
+  cursor: pointer;
+}
+
+.excitedVol::-moz-range-thumb {
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: #4CAF50;
+  cursor: pointer;
+}
+
+.signUpImg {
+  width: 100%;
+}
+
+.signUpImg img {
+  width: 100%;
 }
+
+form input {
+  border: none;
+  border-bottom: 1px solid #BABCBB;
+  height: 30px;
+}
+
+.getStartedBtn {
+  width: 200px;
+  height: 50px;
+  border: none;
+  color: #ffffff;
+  background-color: #62AC7B;
+  margin: 25px 0px;
+  font-size: 18px;
+  border-radius: 10px;
+  font-weight: 600;
+  
+}
+
+.signUpForm h2 .signUpGreen{
+  color: #62AC7B;
+  font-weight: 900;
+}
+
+/* Media queries for form tabelet */
+@media only screen and (min-width: 768px) {
+
+  .signUpContent{
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding: 50px 50px;
+  }
+
+  .signUpImg {
+    width:90%;
+    align-content: center;
+    align-self: center;
+    padding: 20px;
+  }
+
+  .signUpForm h2 {
+    font-size: 40px;
+    text-align: left;
+  }
+} 
+
+/* Media queries for form desktop */
+@media only screen and (min-width: 992px) {
```

Comment:
Try to keep the media queries with the same min-width' separate - now you got a few of these in different places, so check on this